docs/api/_index.md

```diff
@@ -431,6 +433,14 @@ GET /ruler_ring
 
 Displays a web page with the ruler hash ring status, including the state, healthy and last heartbeat time of each ruler.
 
+### Ruler rules
+
+```
+GET /ruler/rules
+```
+
+List all tenant rules that are currently loaded. This endpoint returns a YAML dictionary with all the rule groups for each tenant and `200` status code on success.
```

Comment:
> List all tenant rules that are currently loaded. 

I think this is misleading. Each ruler will have different set of rule groups loaded in memory, but this endpoint will always list them all.